.github/workflows/ci.yaml

```diff
@@ -0,0 +1,26 @@
+name: Dart CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup working Directory
+        run: |
+          sudo mkdir -p /usr/src/extension
+          sudo cp -R ./* /usr/src/extension
+          cd /usr/src/extension
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '17'
```

Comment:
Hmm actually, we're going to need the GHA to have at least a call to
https://github.com/Workiva/gha-store-artifacts and
https://github.com/Workiva/gha-upload-test-reports

which will need https://github.com/Workiva/gha-setup-credentials
and a permission block like
```
permissions:
  contents: read
  id-token: write
```

Reply:
There are no self-hosted runners available for this repo (list [here](https://github.com/Workiva/vs-code-format-on-save/actions/runners?tab=self-hosted)). So I am getting following errors for suggested actions [here](https://github.com/Workiva/vs-code-format-on-save/actions/runs/7498022974/job/20412502946) 

**Error: Unable to resolve action `Workiva/gha-setup-credentials`, repository not found. Unable to resolve action `Workiva/gha-store-artifacts`, repository not found. Unable to resolve action `Workiva/gha-upload-test-reports`, repository not found**

I have tried by updating permissions as well, but that didn't work.

I have used **actions/upload-artifact@v4** (refering from other public repos) and that worked.

Reply:
Ooops, sorry. I didn't notice this was an open source repo so we can use the actions I recommended.